pkgs/top-level/all-packages.nix

```diff
@@ -13208,6 +13208,8 @@ with pkgs;
 
   dex-oidc = callPackage ../servers/dex { };
 
+  doh-proxy = python3Packages.callPackage ../servers/dns/doh-proxy { };
```

Comment:
Please use only `callPackage` and instead pass `python3` as an argument. This allows users to override the python packages specifically for this application.

Reply:
Done, but with `python3Packages` instead of `python3`, so I could access the packages.

Reply:
That's okay. You could also have used `python3.pkgs` instead of `python3Packages`.